### PR TITLE
Remove hardcoded team and team colors values.

### PR DESCRIPTION
**Pull request by jmg**
Remove hardcoded team and team colors values. 
Extracted team colors to a variable and added it to .env file

### CODE REVIEW

File: .env-example [hunk 1]
```diff
@@ -15,4 +15,5 @@ OPENAI_API_KEY  = 1234567890
 
 # Bot constants
 TEAM_NAME = 'Boca Jrs'
+TEAM_COLORS = 'azul y amarrillo'
```
Comment (fergmolina) [inline .env-example]: Change to TEAM_COLORS = 'Azul y Amarillo'

File: bocaGPT.py [hunk 2]
```diff
@@ -62,8 +62,13 @@ def main(event, context):
...
         return (tweets_msg, 200)
 
     except Exception as e:
-        return ("error - " + str(e), 400)
\ No newline at end of file
+        return ("error - " + str(e), 400)
+
+
+if __name__ == "__main__":
+    #Allow running it outside Google Cloud
```
Comment (fergmolina) [inline bocaGPT.py]: Delete comment

**Review by fergmolina — CHANGES_REQUESTED** [review]
Please see comments in env and bocagpt.py

File: .env-example (hunk 1 above)
Comment (jmg) [reply .env-example]: Done!

File: bocaGPT.py (hunk 2 above)
Comment (jmg) [reply bocaGPT.py]: Done!

**Pull request merged**